lib/browser/api/in-app-purchase.js

```diff
@@ -0,0 +1,46 @@
+'use strict'
+
+const binding = process.atomBinding('in_app_purchase')
+const v8Util = process.atomBinding('v8_util')
+
+module.exports = {
+
+  canMakePayments: function() {
+    return binding.canMakePayments();
+  },
+
+  getReceiptURL: function() {
+    return binding.getReceiptURL();
+  },
+
+  purchaseProduct: function(productID, quantity, callback) {
```

Comment:
If you want to allow this method to be called in forms
`purchaseProduct(someId, 2)` and `purchaseProduct(someId, function() {})`,
you will have to check the type of the second argument to decide if it is `quantity` or `callback`.

You can do it like this:
```js
if (typeof quantity == 'function') {
  quantity = 1
  callback = quantity
}
```